src/App.tsx

```diff
@@ -1,24 +1,49 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useCallback, useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { TodoStatus } from './types/TodoStatus';
+import { filterTodos } from './utils/helpers';
+
+import { TodoList } from './components/TodoList';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { NotificationError } from './components/NotificationError';
+
+const USER_ID = 6922;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [todoStatus, setTodoStatus] = useState<TodoStatus>(TodoStatus.ALL);
+  const [error, setError] = useState('');
+
+  const loadTodos = useCallback(async () => {
+    try {
+      const loadedTodos = await getTodos(USER_ID);
+
+      setTodos(loadedTodos);
+    } catch (err) {
+      setError('load');
+      setTimeout(() => {
+        setError('');
+      }, 3000);
+    }
+  }, []);
+
+  useEffect(() => {
+    loadTodos();
+  }, []);
+
+  const visibleTodos = filterTodos(todos, todoStatus);
```

Comment:
will be better to use useMemo hook here, and in dependency arr put todos and todoStatus